docs/src/content/docs/api/tools.mdx

```diff
@@ -7,16 +7,57 @@ import { Badge, Tabs, TabItem } from '@astrojs/starlight/components';
 
-### Two Types of Tools
-Ultravox supports durable and temporary tools. There is no functional difference between them. Durable tools are persisted and can be reused across calls or applications. Temporary tools are created each time you create a new Call and exist exclusively within the context of that call. (Temporary tools aren't visible in the ListTools response for example.)
+## Tools Quickstart
+The fastest way to get started with tools is to pass them in when you call [`Create Call`](../calls/#create-call) in the request body:
 
-Iteration is faster when using temporary tools. Durable tools shine once you have things dialed in, when you want to share tools across multiple applications, or if you have split responsibilities on the team.
+```javascript
+// Request body for creating an Ultravox call with a tool
+{
+  "model": "fixie-ai/ultravox-70B",
+  "systemPrompt": ...
+  "selectedTools": [
+    "temporaryTool": {
+      "nameOverride": "stock_price",
+      "description": "Get the current stock price for a given symbol",
+      "dynamicParameters": [
+        {
+          "name": "symbol",
+          "location": "PARAMETER_LOCATION_QUERY",
+          "schema": {
+            "type": "string",
+            "description": "Stock symbol (e.g., AAPL for Apple Inc.)"
+          },
+          "required": true
+        }
+      ],
+      "http": {
+        "baseUrlPattern": "https://api.stockmarket.com/v1/price",
+        "httpMethod": "GET"
+      }
+    }
+  ]
+}
+```
+We call these temporary tools.
 
-### Creating Tools
+## Temporary Tools
+Temporary tools are created each time you create a new Call and exist exclusively within the context of that call. (Temporary tools aren't visible in the [List Tools](./#list-tools) response for example.)
+
+Iteration is faster when using temporary tools.
+
+### Creating Temporary Tools
 Temporary tools are defined and passed in the request body of the [`Create Call`](../calls/#create-call) endpoint.
 
+## Durable Tools
+In addition to temporary tools, Ultravox supports the creation of durable tools. There is no functional difference between durable and temporary tools.
+
+Durable tools are persisted and can be reused across calls or applications. They shine once you have things dialed in, when you want to share tools across multiple applications, or if you have split responsibilities on the team.
+
+### Creating Durable Tools
 You create durable tools either by uploading an OpenAPI spec or via the request body in the [`Create Tool`](./#create-tool) endpoint. Your OpenAPI spec must be either `json` or `yaml` format.
 
+The `/tools` endpoint in the Ultravox API is for working with durable tools.
+
```

Comment:
Maybe a quick section on how to use durable tools?

### Using Durable Tools
To use a durable tool in a call, set the toolName or toolId field instead of temporaryTool.  For example:

```javascript
// Request body for creating an Ultravox call with a tool
{
  "model": "fixie-ai/ultravox-70B",
  "systemPrompt": ...
  "selectedTools": [
    "toolName": "stock_price",
  ]
}

Reply:
good idea